src/app/components_ngrx/toolbar-panel/toolbar-panel.component.html

```diff
@@ -46,12 +46,12 @@
         </aside>
         <div>
           <input class="form-control ng-pristine ng-valid ng-touched"
-          (keyup.enter)="saveFilters(filterSaveInp.value)"
-          #filterSaveInp />
+          (keyup.enter)="saveFilters(filterSaveInp,filterSaveInp.value)"
```

Comment:
Missing space after the `,`

Reply:
If we are already passing `filterSaveInp` what is the need of passing a property of it separately?